connpool: close unused connections automatically (#11548)
tarantool/tarantool: changelogs/unreleased/connpool-close-unused-conns.md

Reviewer: Nit: No 'part of' or some similar link in the second commit.

## feature/connpool

> * `experimental.connpool` now automatically closes unused connections opened by
>   `connpool.call()` and `connpool.filter()`. This deadline can be configured
>   using the `connpool.set_idle_timeout()` method or the `connpool.idle_timeout`
>   configuration option.

Reviewer 2: ```suggestion
* `experimental.connpool` now automatically closes unused connections opened by
  `connpool.call()` and `connpool.filter()` upon the timeout configured
  using the `connpool.set_idle_timeout()` method or the `connpool.idle_timeout`
  configuration option.
```

Author: Thank you!
